### PR TITLE
Added BB spawn

### PR DESCRIPTION
**Pull request by BorkBorkGoesTheCode**
Added BBs to ammo_rifle_common.

### CODE REVIEW

**Comment by mugling** [comment]
Not appropriate in those item groups

**Comment by BorkBorkGoesTheCode** [comment]
ammo_pistol_common?

**Comment by mugling** [comment]
Neither. You won't find BB's in most of the places those item groups are used. Currently those items are found in houses which is fine.

**Comment by BorkBorkGoesTheCode** [comment]
Big box stores carry BBs, more than regular ammo.

**Comment by mugling** [comment]
That's fine but we don't have those stores in game and if you add it to those groups they're going to turn up in odd places

**Comment by Coolthulhu** [comment]
Could have its own spawn point in gunshops.

**Comment by mugling** [comment]
Or better still in garages and other such locations

**Comment by mugling** [comment]
> supplies_spares_vehicle?

Yes that could work

File: data/json/itemgroups/ammo.json [hunk 1]
```diff
@@ -138,7 +138,8 @@
       [ "270", 25 ],
       [ "3006", 25 ],
       [ "308", 25 ],
-      [ "762_m43", 10 ]
+      [ "762_m43", 10 ],
+      [ "bb", 80 ]
```
Comment (mugling) [inline data/json/itemgroups/ammo.json]: Still need to revert the earlier change

**Comment by BorkBorkGoesTheCode** [comment]
Removed

**Pull request merged**